x-pack/legacy/plugins/snapshot_restore/public/app/components/restore_snapshot_form/steps/step_logistics.tsx

```diff
@@ -156,89 +171,146 @@ export const RestoreSnapshotStepLogistics: React.FunctionComponent<StepProps> =
+                          >
+                            <FormattedMessage
+                              id="xpack.snapshotRestore.restoreForm.stepLogistics.indicesToggleListLink"
+                              defaultMessage="Select indices"
+                            />
+                          </EuiLink>
+                        </EuiFlexItem>
+                      </EuiFlexGroup>
+                    )
                   }
                   helpText={
-                    <FormattedMessage
-                      id="xpack.snapshotRestore.restoreForm.stepLogistics.selectIndicesHelpText"
-                      defaultMessage="{count} {count, plural, one {index} other {indices}} will be restored. {selectOrDeselectAllLink}"
-                      values={{
-                        count: restoreIndices && restoreIndices.length,
-                        selectOrDeselectAllLink:
-                          restoreIndices && restoreIndices.length > 0 ? (
-                            <EuiLink
-                              onClick={() => {
-                                indicesOptions.forEach((option: Option) => {
-                                  option.checked = undefined;
-                                });
-                                updateRestoreSettings({ indices: [] });
-                                setCachedRestoreSettings({
-                                  ...cachedRestoreSettings,
-                                  indices: [],
-                                });
-                              }}
-                            >
-                              <FormattedMessage
-                                id="xpack.snapshotRestore.restoreForm.stepLogistics.deselectAllIndicesLink"
-                                defaultMessage="Deselect all"
-                              />
-                            </EuiLink>
-                          ) : (
-                            <EuiLink
-                              onClick={() => {
-                                indicesOptions.forEach((option: Option) => {
-                                  option.checked = 'on';
-                                });
-                                updateRestoreSettings({ indices: [...snapshotIndices] });
-                                setCachedRestoreSettings({
-                                  ...cachedRestoreSettings,
-                                  indices: [...snapshotIndices],
-                                });
-                              }}
-                            >
-                              <FormattedMessage
-                                id="xpack.snapshotRestore.restoreForm.stepLogistics.selectAllIndicesLink"
-                                defaultMessage="Select all"
-                              />
-                            </EuiLink>
-                          ),
-                      }}
-                    />
+                    selectIndicesMode === 'list' ? (
```

Comment:
I don't have a good alternative atm, but i'm not sure if it would have been immediately clear to me that the `Custom pattern`/`Show indices` link is used to toggle views if i wasn't familiar with the PR changes.

Reply:
@gchaps Could you help me with the wording for the toggle links?

Currently they are: `Custom pattern` and `Select indices`. You can see them in the screenshots in PR description. I agree with Alison that the current wording doesn't correlate very strongly to the fact that the links themselves toggle between a text input view and a selectable list view.

Maybe they could be: `Enter custom pattern` and `Select from list` instead? Help!! 🙂

Reply:
I had the same thought when I looked at the copy. How about "Use index pattern" and "Choose individual indices"?

In terms of prior art, in the Rollup Job Wizard we use "Create cron expression" and "Create basic interval" to switch between two UIs for creating what eventually becomes a cron expression, though I'm not convinced "Create" is the right verb to use here. I think I would prefer "Use".

![image](https://user-images.githubusercontent.com/1238659/61826489-59779200-ae17-11e9-826e-1f782e7652f3.png)

![image](https://user-images.githubusercontent.com/1238659/61826493-5aa8bf00-ae17-11e9-98e9-6c8298ab4f66.png)

Reply:
@jen-huang Based on the suggestion from @cjcenizal , what about:

Use index pattern
Select index

Reply:
Thanks @gchaps! Is there a reason to use "Select index" instead of "Select indices"? Is singular better than plural in these cases?

Reply:
@cjcenizal If the user can select more than one index, then "Select indices" is better.